server/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/EjeaFetchingCachedOutputsFromDatabaseSpec.scala

```diff
@@ -1,85 +0,0 @@
-package cromwell.engine.workflow.lifecycle.execution.ejea
```

Comment:
Do we have to get rid of those too ? Those EJEA unit test despite being hard to read are valuable I think because they test a lot of corner cases that can't be tested through centaur.

Reply:
@Horneth I don't think there should be any reason for a test spec to have the "run a workflow with full cromwell" though, so those tests should be representable via some other mechanism. Do you disagree in this case, or is your complaint more like w/ the single workflow where we need to find another way to test that?

Reply:
Those `EjeaDoingSomething` tests are not running workflows, they're testing how the EJEA responds to different messages while being in specific states. It tests that we didn't forget a `case ...` in the FSM and that things won't stay stuck.

Reply:
it's possible that methods unrelated to "run a WDL" got bundled into `CromwellTestKitSpec` because it sounded a bit like `AkkaTestKit`

Reply:
How about we sit down and go through this, come up with a scheme for preserving anything needing preserving via a means other than CTKS and then decide on a case by case basis if it makes sense to do that now or bit by bit